alf/networks/projection_networks.py

```diff
@@ -777,3 +777,78 @@ def forward(self, inputs, state=()):
                 reinterpreted_batch_ndims=len(self._output_shape) - 1), state
         else:
             return dist_cls(logits=logits), state
+
+
+@alf.configurable
+class MixtureProjectionNetwork(Network):
+    """A projection network that outputs MixtureSameFamily distributions.
+
+    The output distribution consists of 2 parts:
+
+    1. A categorical distribution for each of the component.
+    2. A components distribution of ``num_components`` replicas.
+    
+    """
+
+    def __init__(
+            self,
+            input_size: int,
+            action_spec: TensorSpec,
+            num_components: int,
+            component_ctor: Callable[[int, TensorSpec], Network],
+            mixture_ctor: Callable[[int, BoundedTensorSpec],
+                                   Network] = CategoricalProjectionNetwork,
+            name: str = "mix_proj_net"):
+        """Constructs an instance of MixtureProjectionNetwork.
+
+        Args
+
+            input_size: the input vector size
```

Comment:
Minor format issue: Args -> Args:    and  can remove line 805

Reply:
Fixed. Thanks!